cmd/lotus-chainwatch/storage.go

```diff
@@ -294,6 +294,30 @@ create table if not exists miner_info
 		primary key (miner_id)
 );
 
+/*
+* captures chain-specific power state for any given stateroot
+*/
+create table if not exists chain_power
+(
+	stateroot text not null
```

Comment:
I _think(?)_ this needs a comma at the end.
Other table call this `state_root`, we should here too for consistency.

Reply:
The comma is on L304 since I'm concatinating the pk constraint onto the column.